pkg/migrator/reader.go

```diff
@@ -71,10 +89,18 @@ func NewGroupMessageReader(db *sql.DB) *GroupMessageReader {
 		ORDER BY id ASC
 		LIMIT $2
 	`
+	queryHeight := `
+		SELECT id
+		FROM group_messages
+		WHERE is_commit = false
+		ORDER BY id DESC
+		LIMIT 1
+	`
```

Comment:
I am trying really hard to avoid having SQL queries sprinkled throughout the code. If its tests, its OK. But for all production code, I want it in the SQLC stuff. It makes it very hard to track changes if its all over the place. I had to fight a bunch of gateway_envelope queries scattered through the code base.